client/src/stylesheets/index.css

```diff
@@ -124,7 +124,6 @@ a:hover {
 }
 
 .define-form {
```

Comment:
I could be wrong, but I think this css could be removed entirely. Have you tried commenting it out and checking the UI appearance?

Reply:
Good catch. Removed it.